go.mod

```diff
@@ -6,22 +6,22 @@ require (
 	github.com/Microsoft/go-winio v0.6.1
 	github.com/apex/log v1.9.0
 	github.com/buildpacks/imgutil v0.0.0-20230626185301-726f02e4225c
-	github.com/buildpacks/lifecycle v0.17.2
-	github.com/docker/cli v24.0.7+incompatible
+	github.com/buildpacks/lifecycle v0.17.1
+	github.com/docker/cli v24.0.6+incompatible
```

Comment:
@jericop as we talked, just rebase your branch with the latest from main because most of your changes in libraries are reverting the last updates. Apart from that, imgutil changes looks good to me.

Reply:
Sorry about that. VScode was configured to automatically update the go.mod file every time it gets saved.